sql/driver/driver_test.go

```diff
@@ -81,7 +81,7 @@ func TestCreateDatabase(t *testing.T) {
 	if _, err := db.Exec(`CREATE DATABASE foo`); err != nil {
 		t.Fatal(err)
 	}
-	if _, err := db.Exec(`CREATE DATABASE foo`); !isError(err, "database .* already exists") {
+	if _, err := db.Exec(`CREATE DATABASE foo`); !isError(err, "\\*structured.DatabaseDescriptor .* already exists") {
```

Comment:
I didn't want to pass "database" or "table" to the shared functions just for the purposes of the error message, so I'm printing the type.